xbmc/addons/gui/GUIDialogAddonInfo.cpp

```diff
@@ -767,33 +770,42 @@ void CGUIDialogAddonInfo::BuildDependencyList()
 
     if (!addonInstalled)
     {
-      // after pushing the install button the dependency install dialog will
-      // pop up only if non-module dependencies are going to be installed or
-      // dependencies are unavailable. the latter is for informational purposes
-
-      if (showAllDependencies || !addonAvailable ||
-          addonAvailable->MainType() != ADDON_SCRIPT_MODULE)
+      // after pushing the install button the dependency install dialog
+      // will be opened only if...
+      // - dependencies are unavailable (for informational purposes) OR
+      // - the dependency is not a script/module                     OR
+      // - the script/module is not available at an official repo
+
```

Comment:
This feel off with the empty line between the comment and the line it refers to. Maybe an empty line before the comment and no empty line after?

Reply:
yes, done for all